assets/component-newsletter.css

```
@@ -44,11 +48,16 @@
 
 .newsletter-form__button {
   width: 4.4rem;
-  margin: 0;
-  right: var(--inputs-border-width);
+  /* margin: 0; */
+  /* right: var(--inputs-border-width);
   top: 0;
-  height: 100%;
+  height: 100%; */
   z-index: 2;
+  display: flex !important;
```

```suggestion
  display: flex;
```


We should avoid `!important` at all costs. Why did you use it in this instance? Maybe you needed to apply for the property elsewhere. When I see the preview link I can evaluate further.